### ISSUE TITLE
View Versions Unavailable in Past Versions

### ISSUE THREAD

**Issue by ZachGerth** [tsgrp/hpi#934]
In Alfresco, if you view a version of a form that is older than the current version via. the View Versions button, the View Versions button no longer appears.
Latest Version:
![image](https://cloud.githubusercontent.com/assets/14186924/12039345/44949b80-ae25-11e5-899b-2ee13066203d.png)

Previous Version:
![image](https://cloud.githubusercontent.com/assets/14186924/12039384/9a5ccb32-ae25-11e5-99f8-91e8a20427df.png)

It looks like this isn't just restricted to forms, but other documents as well.

I couldn't replicate this in Dctm.

**Comment by gsteimer** [comment]
I don't think this is an issue with the code, more so with how Alfresco does versioning differently than how Dctm or HBase work.  I'll mark this as upon client request though in case a client does want us to find a workaround.

**Comment by ZachGerth** [comment]
It looks like this is specific to minor versions; between major versions, the button remains visible.
